docs/zh/sql-reference/sql-functions/array-functions/array_sortby.md

```diff
@@ -46,6 +47,8 @@ array_sortby(<lambda function>, array0 [, array1...])
 - `array0`：需要排序的数组，支持的数据类型为 ARRAY，或者 `null`。数组中的元素必须为可排序的元素。
 - `array1`：用于排序的键值数组，支持的数据类型为 ARRAY，或者 `null`。
 - `lambda_function`：lambda 函数，用于生成排序键值数组。
```

Comment:
Maybe we should give a note that :
- If `lambda_function` exists, we will only use the result of `lambda_function` to sort.
- Otherwise, will use the non-first arrays to sort.